client/src/controllers/ClipboardController.ts

```diff
@@ -0,0 +1,35 @@
+import { Controller } from '@hotwired/stimulus';
+
+export class ClipboardController extends Controller<HTMLElement> {
+  static targets = ['value'];
+
+  declare valueTarget:
+    | HTMLInputElement
+    | HTMLTextAreaElement
+    | HTMLSelectElement;
+
+  copy() {
+    this.dispatch('start');
```

Comment:
```suggestion
    const event = this.dispatch('copy');
    if (event.defaultPrevented) return;
```

Let's rename this as `copy`, it's probably a bit nicer than `start`.

Additionally with a one line defaultPrevented check we can enable a much more flexible usage of this in the future.